### ISSUE TITLE
[Defect]: 

### ISSUE THREAD

**Issue by damen-dotcms** [damen-dotcms/issue-test#28]
### Proposed Objective

Accessibility (WCAG)

### Proposed Priority

Priority 1 - Show Stopper

### dotCMS Version

TEST

### Problem Statement

TEST

### Steps to Reproduce

TEST

### Acceptance Criteria

TEST

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.

TEST

### Assumptions & Initiation Needs

TEST

### Sub-Tasks

TEST

### Estimates

TEST
